msys2-runtime/0019-strace.cc-Don-t-set-MSYS-noglob.patch

```diff
@@ -50,12 +50,17 @@ index 0c8d75e..f7c98bd 100644
 +  See quoted () comment:
 +  "This must have been run from a Windows shell, so preserve
 +     quotes for globify to play with later."
-+
+ 
    const char *cygwin_env = getenv ("MSYS");
    const char *space;
-   if (cygwin_env)
-@@ -360,6 +378,7 @@ create_child (char **argv)
-   char *newenv = (char *) malloc (sizeof ("MSYS=noglob") + strlen (space) + strlen (cygwin_env));
+ 
+-  if (cygwin_env && strlen (cygwin_env) <= 256) /* sanity check */
++  if (cygwin_env && strlen (cygwin_env) <= 256) // sanity check
+     space = " ";
+   else
+     space = cygwin_env = "";
+@@ -364,6 +381,7 @@ create_child (char **argv)
+ 				  + strlen (space) + strlen (cygwin_env));
    sprintf (newenv, "MSYS=noglob%s%s", space, cygwin_env);
    _putenv (newenv);
 +*/
```

Comment:
@dscho there are no spaces before `+` in lines 56-63 (compare it to 50-52).
Also look at the original patch https://github.com/Alexpux/MSYS2-packages/blob/master/msys2-runtime/0019-strace.cc-Don-t-set-MSYS-noglob.patch